tests/cirq_qir/test_cirq_preprocess.py

```diff
@@ -40,17 +95,17 @@ def test_convert_gridqubits_to_linequbits(gridqubit_circuit):
     linequbit_circuit = preprocess_circuit(gridqubit_circuit)
     for qubit in linequbit_circuit.all_qubits():
         assert isinstance(qubit, cirq.LineQubit), "Qubit is not a LineQubit"
-    assert np.allclose(
-        linequbit_circuit.unitary(), gridqubit_circuit.unitary()
+    _assert_allclose_up_to_global_phase(
```

Comment:
`qbraid` is already a test dependency so you can just use `qbraid.interface.assert_allclose_up_to_global_phase` instead of copying over those same funcs